python/openai/openai_frontend/routers/chat.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
I wonder if the `routers` should be part of the frontend specialization.

Reply:
Can you elaborate? Just moved into the `frontend` folder? Or including the routes as part of the class/implementation?

Let me know if you have a specific preferred change.

Reply:
`frontend/fastapi/routers/...`

Reply:
done